src/cache/RedisQueryResultCache.ts

```diff
@@ -60,14 +60,12 @@ export class RedisQueryResultCache implements QueryResultCache {
             } else {
                 this.client = new this.redis();
             }
-
         } else if (this.clientType === "ioredis/cluster") {
-            if (cacheOptions && cacheOptions.options) {
-                this.client = new this.redis.Cluster(cacheOptions.options);
+            if (cacheOptions && cacheOptions.options && cacheOptions.options.startupNodes) {
```

Comment:
`options` argument is not required according to source code from `ioredis` so do not have to check `cacheOptions.options` existence.
https://github.com/luin/ioredis/blob/fbcf692f5de8c653139e4b053c6356ab6d67e463/lib/cluster/index.ts#L65

Reply:
Nice enhancement @scvgoe!

This looks good to me. The optional value is actually cacheOptions.options.options.  

cacheOptions.options is the typeORM options object with the general Cluster Ioredis properties including the mandatory one: startupNodes. So in the case of Cluster Redis its always required. 

I have to say that: cacheOptions.options.options looks pretty weird and maybe confusing, as it has plenty of options. Though I have no better suggestion than changing the name of the property to clusterOptions. Yet this is none important, so ignore this if you want :)

Reply:
Sry man :-) did not realize it because of so much "options".

Reply:
Hahaha, exactly!

Reply:
So just add recordt to `CHANGELOG.md` and it is ready to merge.

Reply:
Thank you for your reviewing!!